### PR TITLE
Fix#261 Replace vars with lets

### PR DESCRIPTION
**Pull request by haichuan0424**
All vars in price.js can be replaced with lets since they are used within the declaration scope. However, some of the variable names are exact the same for function parameters and local variables, which might cause confusion. You might want to file an issue to rename some of them. 

For file purge.js, I moved variable "newamount" declaration outside the if/else block so it can be declared with let.  

In conclusion, the fix replaced all vars with lets. Only two files price.js and purge.js are involved. There is no "var" left in any file. Let me know if any issues or further changes needed.

### CODE REVIEW

**Comment by nikooo777** [comment]
Thank you, this looks good to me. I haven't checked the codebase for other vars yet, but as soon as I do so I'll merge this.

I'll then consider if the renaming issue is worth filing

**Comment by Coolguy3289** [comment]
Just confirmed through Github repo search that vars only existed in the two files mentioned above. LGTM.

**Comment by tzarebczan** [comment]
@haichuan0424 , thanks for the PR! 

Can we show you some [appreciation](https://lbry.com/faq/appreciation) for the contribution? 

Also, we're giving away some [Hacktoberfest](https://lbry.com/news/hacktoberfest-2019) bonuses and goodies for this month. We'll get it all figured out after you shoot us an email after this is reviewed/merged.

**Comment by Coolguy3289** [comment]
Already merged

**Comment by haichuan0424** [comment]
@tzarebczan 
Glad I could help! It would be great to get some bonuses.
